functions.Rmd

```diff
@@ -14,6 +14,24 @@ row_adder()
 permutation()
 ```
 
+## Anonymous functions
+
+If using an anonymous or inline function (i.e. a function that hasn't been given a name by assigning it to a variable), prefer the new lambda syntax: `\(x) x + 1`.
```

Comment:
"If using _short_ anonymous or inline functions"

I think that would also help